js/components/Message/MessageRepeat.js

```diff
@@ -2,8 +2,8 @@ import React, { PropTypes } from "react";
 
 function MessageRepeat(props) {
   return (
-    <span value="1" className="message-repeats">
-      // @TODO implement
+    <span value="{props.repeats}" className="message-repeats">
```

Comment:
If this were the final thing, we should have a propTypes definition for repeats, but eh... that can happen later.